addons/l10n_my_edi_extended/models/account_move_line.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from odoo.addons.l10n_my_edi.models.product_template import CLASSIFICATION_CODES_LIST
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    # ------------------
+    # Fields declaration
+    # ------------------
+
+    l10n_my_edi_classification_code = fields.Selection(
+        string="Malaysian classification code",
+        selection=CLASSIFICATION_CODES_LIST,
+        compute="_compute_l10n_my_edi_classification_code",
+        store=True,
+        readonly=False,
+        copy=False,
+    )
+
+    # --------------------------------
+    # Compute, inverse, search methods
+    # --------------------------------
+
+    @api.depends("product_id.product_tmpl_id.l10n_my_edi_classification_code")
```

Comment:
This can be a perf issue: when changing the code on a product, it will try to update all the moves

```suggestion
    @api.depends("product_id.product_tmpl_id")
```